2023/assignments/fiesty-cushion/fiesty-cushion.md

```diff
@@ -0,0 +1,151 @@
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+    <a href="https://youtu.be/dQw4w9WgXcQ" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/github.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+       <a href="https://youtu.be/dQw4w9WgXcQ" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/instagram.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+
+</p>
+
+<br>
+
+## <p align='center'>Learned Commands</p>
+
+1.  `git status`
+
+    **Description**:
+
+        Shows the current status of your working directory and staging area, including tracked, modified, and untracked files.
+
+    **Examples**:
+
+        $ git status
+    
+2.  `git clone <repository URL>`
+
+    **Description** :
+
+        Copies an existing repository from a remote source (GitHub) to your local machine.
+
+    **Examples**:
+
+        $ git clone https://github.com/Fiesty-Cushion/First-Contribution.git
+
+3.  `git pull`
+
+    **Description** :
+
+        Fetches and merges the changes from remote repository to your local machine.
+
+    **Examples**:
+
+        $ git pull
+
+4.  `git push`
+
+    **Description** :
+
+        Pushes the commits from your local branch to the remote repository, updating it with your changes.
+
+    **Examples**:
+
+        $ git push
+
+5.  `git branch`
+
+    **Description** :
+
+        Lists all branches in the repository and highlights the branch we are currently working on.
+
+    **Examples**:
+
+        $ git branch
+
+6.  `git checkout <branch>`
+
+    **Description** :
+
+        Switches to the specified branch.
+
+    **Examples**:
+
+        $ git checkout main
+
+7.  `git remote add <name> <URL>`
+
+    **Description** :
+
+        Links your local repository to a remote repository with the given name and URL.
+
+    **Examples**:
+
+        $ git remote add origin https://github.com/Fiesty-Cushion/First-Contribution.git
+
+8.  `git log`
+
+    **Description** :
+
+        Displays a chronological list of commits in the current branch along with commit messages and authors.
+
+    **Examples**:
+
+        $ git log || git log -<n> || git log --oneline
+
+9.  `git merge <branch>`
+
+    **Description** :
+
+        Merges the commits from specified branch into your current branch.
+
+    **Examples**:
+
+        $ git merge major-changes
+
+10. `git diff`
+
+     **Description** :
+
+         Shows the differences between the working directory and the staging area.
+
+     **Examples**:
+
+         $ git diff
+
```

Comment:
@Fiesty-Cushion remove the trailing line breaks, just one line break after `$ git diff`